### DISCUSSION TITLE
Possible to have sql generated default values?

### DISCUSSION THREAD

**Discussion by ChaseLewis** [SeaQL/sea-orm#1289] [Q&A]
Most ORM's have a means by which to specify sql generated default values. For uuid's  this would be "uuid_generate_v4()" which is a method. Trying to figure out if there is a way to do this as this is a pretty low level requirement for an ORM.

**Comment by billy1624** [comment]
Hey @ChaseLewis, welcome!! I think the default value should be set during creation of the db schema.

**Reply by ChaseLewis** [reply]
@billy1624 I get it would be set in the schema. My question more so is how does one set a default value generated by the database in the schema.

**Reply by billy1624** [reply]
Insert the row as usual and leave the column with default value as null. DB will fill the column for you.
